client.js

```diff
@@ -121,27 +121,36 @@ RingpopClient.prototype._request = function _request(opts, endpoint, head, body,
         return;
     }
 
-    this.subChannel.waitForIdentified({
-        host: opts.host
-    }, function onIdentified(err) {
-        if (err) {
-            callback(err);
-            return;
-        }
+    try {
+        request();
+    } catch(e) {
+        callback(e);
+        return;
+    }
```

Comment:
Applications that *do not* continue on uncaught is not something we should support. 

If you use ringpop; you continue on uncaught.

@jwolski we should definitely be more careful about host port assertions. we can document what a host:port is in the tchannel docs.